src/icons/css/sprite.css

```diff
@@ -0,0 +1,506 @@
+.hyicon {
+    font-size: 10px;
+}
+
+.hyicon:before {
+    content:' ';
+    vertical-align:middle;
+    display: inline-block;
+    background-image: url("../svg/sprite.svg");
+    background-repeat: no-repeat;
+    background-size: 13.2em 712.8em;
+}
+
+.no-svg .hyicon:before {
+    background-image: url("../svg/sprite.png");
```

Comment:
No PNGs are being output. Do we even need the PNGs? If not, these fallback rules should be removed.